admin_manual/installation/php_configuration.rst

```diff
@@ -0,0 +1,132 @@
+* PHP module ctype
+* PHP module curl
+* PHP module dom
+* PHP module fileinfo (included with PHP)
+* PHP module filter (only on Mageia and FreeBSD)
+* PHP module GD
+* PHP module hash (only on FreeBSD)
+* PHP module JSON (included with PHP >= 8.0)
+* PHP module libxml (Linux package libxml2 must be >=2.7.0)
+* PHP module mbstring
+* PHP module openssl (included with PHP >= 8.0)
+* PHP module posix
+* PHP module session
+* PHP module SimpleXML
+* PHP module XMLReader
+* PHP module XMLWriter
+* PHP module zip
+* PHP module zlib
+
+Database connectors (pick the one for your database:)
+
+* PHP module pdo_sqlite (>= 3, usually not recommended for performance reasons)
+* PHP module pdo_mysql (MySQL/MariaDB)
+* PHP module pdo_pgsql (PostgreSQL)
+
+*Recommended* packages:
+
+* PHP module bz2 (recommended, required for extraction of apps)
+* PHP module intl (increases language translation performance and fixes sorting
+  of non-ASCII characters)
+* PHP module sodium (for Argon2 for password hashing. bcrypt is used as fallback, but if passwords were hashed with Argon2 already and the module is missing, your users can't log in.)
+
+Required for specific apps:
+
+* PHP module ldap (for LDAP integration)
+* PHP module smbclient  (SMB/CIFS integration, see
+  :doc:`../configuration_files/external_storage/smb`)
+* PHP module ftp (for FTP storage / external user authentication)
+* PHP module imap (for external user authentication)
+* PHP module bcmath (for passwordless login)
+* PHP module gmp (for passwordless login)
+
+Recommended for specific apps (*optional*):
+
+* PHP module gmp (for SFTP storage)
+* PHP module exif (for image rotation in pictures app)
+
+For enhanced server performance (*optional*) select one of the following
+memcaches:
+
+* PHP module apcu (>= 4.0.6)
+* PHP module memcached
+* PHP module redis (>= 2.2.6, required for Transactional File Locking)
+
+See :doc:`../configuration_server/caching_configuration` to learn how to select
+and configure a memcache.
+
+For preview generation (*optional*):
+
+* PHP module imagick
+* avconv or ffmpeg
+* OpenOffice or LibreOffice
+
+.. note::
+   If the preview generation of PDF files fails with a "not authorized" error message, you must adjust the imagick policy file.
+   See https://cromwell-intl.com/open-source/pdf-not-authorized.html
+
+For command line processing (*optional*):
+
+* PHP module pcntl (enables command interruption by pressing ``ctrl-c``)
+
+.. note::
+   You also need to ensure that ``pcntl_signal`` and ``pcntl_signal_dispatch`` are not disabled
+   in your php.ini by the ``disable_functions`` option.
+
+For command line updater (*optional*):
+
+* PHP module phar (upgrades Nextcloud by running ``sudo -u www-data php /var/www/nextcloud/updater/updater.phar``)
+
+ini values
+----------
+
+The following ini settings should be adapted if needed for Nextcloud:
+
+* ``apc.enable_cli``: See :doc:`../configuration_server/caching_configuration`
```

Comment:
Not sure, I think there is no capital after `:` in this case:
```suggestion
* ``apc.enable_cli``: see :doc:`../configuration_server/caching_configuration`
```

Reply:
Do you mean all lines should have no capitalization?

Reply:
Yes